database_cleanup/views/purge_menus.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purge_menus_view" model="ir.ui.view">
+        <field name="model">cleanup.purge.wizard.menu</field>
+        <field name="inherit_id" ref="form_purge_wizard" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <data />
+        </field>
+    </record>
+
+    <record id="action_purge_menus" model="ir.actions.server">
+        <field name="name">Purge menus</field>
+        <field name="type">ir.actions.server</field>
+        <field name="state">code</field>
+        <field name="model_id" ref="database_cleanup.model_cleanup_purge_wizard_menu" />
+        <field name="code">
+            action = env.get('cleanup.purge.wizard').get_wizard_action()
```

Comment:
```suggestion
            action = env.get('cleanup.purge.wizard.menu').get_wizard_action()
```

Reply:
I tested in local environment and get error "You are not allowed to create 'Purge stuff' (cleanup.purge.wizard) records." when click "Purge absolete menu entries". I think you should use the model of current menu.